DisCatSharp.ApplicationCommands/ApplicationCommandsConfiguration.cs

```diff
@@ -39,42 +39,47 @@ public class ApplicationCommandsConfiguration
 		public IServiceProvider ServiceProvider { internal get; set; }
 
 		/// <summary>
-		/// <para>Sets whether to enable default help command.</para>
+		/// <para>This option enables the default help command.</para>
 		/// <para>Disabling this will allow you to make your own help command.</para>
-		/// <para>
-		/// </para>
 		/// <para>Defaults to true.</para>
 		/// </summary>
 		public bool EnableDefaultHelp { internal get; set; } = true;
 
 		/// <summary>
-		/// Debugs the startup.
-		/// </summary>
-		public bool DebugStartup { internal get; set; } = false;
-
-		/// <summary>
-		/// Enable localization features.
+		/// This option enables the localization feature.
 		/// </summary>
 		public bool EnableLocalization { internal get; set; } = false;
 
-		/// <summary>
-		/// Manual override.
-		/// <note type="warning">DO NOT USE THIS!</note>
-		/// </summary>
-		public bool ManualOverride { internal get; set; } = false;
-
 		/// <summary>
 		/// Automatically defer all responses.
 		/// <note type="note">If you enable this, you can't use CreateResponse. Use EditResponse instead.</note>
 		/// </summary>
 		public bool AutoDefer { internal get; set; } = false;
 
 		/// <summary>
-		/// Checks through all guilds.
+		/// <para>This option informs the module to check through all guilds whether the <see target="_blank" alt="Application Commands Scope" href="https://discord.com/developers/docs/topics/oauth2#shared-resources-oauth2-scopes">application.commands</see> scope is set.</para>
```

Comment:
Linebreak pls?

Reply:
Looks like this in the end:
![2022-05-24_03h11_15](https://user-images.githubusercontent.com/14029133/169928615-86eefe20-13e7-4509-87d0-92e23445451e.png)